spatialdata/temp/bug_dask_zarr.py

```diff
@@ -0,0 +1,58 @@
+##
```

Comment:
what is this?

Reply:
It's to use [*Python cell mode* in PyCharm](https://plugins.jetbrains.com/plugin/7858-pycharm-cell-mode). Basically it allows to send code blocks to an open console/debug console. It gives the advantages of Jupyter-like cells without having to abandon `.py` files. I sometimes use it while debugging, can be deleted.